contracts/epochs/FlowEpoch.cdc

```diff
@@ -40,27 +40,27 @@ import FlowFees from 0xFLOWFEESADDRESS
 // indicates that the participants in the next epoch failed to complete the set up
 // procedure, which is a critical failure and will cause the chain to halt.
 
-pub contract FlowEpoch {
+access(all) contract FlowEpoch {
 
-    pub enum EpochPhase: UInt8 {
-        pub case STAKINGAUCTION
-        pub case EPOCHSETUP
-        pub case EPOCHCOMMIT
+    access(all) enum EpochPhase: UInt8 {
+        access(all) case STAKINGAUCTION
+        access(all) case EPOCHSETUP
+        access(all) case EPOCHCOMMIT
```

Comment:
Dang, every case in an enum needs its own access modifier? 😮

Reply:
yeah, a little awkward